components/service-waiter/cmd/component.go

```diff
@@ -107,6 +134,9 @@ func waitPodsImage(ctx context.Context) error {
 			}
 			return ctx.Err()
 		default:
+			if shouldSkipComponentWaiter(ctx) {
+				return nil
```

Comment:
Log for this specific case woudl be great, too!
Afterwards, we will always 100% understand which branch we took based on the log messages. :+1: